### DISCUSSION TITLE
HP 8470p

### DISCUSSION THREAD

**Discussion by kajfas** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#40] [Q&A]
Hello,

I am successful installed Big Sur on HP Elitebook 8470p i5 with your Package without problem. 

I would like to ask you, If there is some solution how to speedup the booting to Mac OS. From choosing the OS to enter the password, it takes about 75 sec. in the Verbose mode(HideVerbose checked). The Clover time on Catalina was about 20 sec.

Thank you for your good work.
Thanks for answer.

**Comment by chris1111** [comment]
Thanks
Post a video of Boot verbose mode here

**Comment by kajfas** [comment]
Hi,

Its installed on SSD Drive with 2 partitions. Second is for Windows, but Its clean now.

https://drive.google.com/file/d/1jbSKYNcvxKkBtY9U89O54Baj2iiSzH4B/view?usp=sharing

Thanks

**Comment by chris1111** [comment]
Probably the issue is the two partition ?
I have Windows UEFI installed on another SSD on my Cady and the boot time is **25 Second** 
😀 ☞ [See Video ](https://www.dropbox.com/s/2hhsly6ji5ahzht/BOOT-OC.MOV?dl=0)

**Comment by chris1111** [comment]
😀 ☞ [See Video ](https://www.dropbox.com/s/2hhsly6ji5ahzht/BOOT-OC.MOV?dl=0)

**Comment by kajfas** [comment]
Hi,

sorry for late respond, but I broke my adapter cable in bag and wait for new adapter.

When I merge the partitions, the time was going down about 10-15 sec. 
But now, I found the problem, when I digging in config.plist. 

Kernel -} Patch -} Enable TRIM for SSD - uncheck

Booting time is now around 25 second.
